File: sdk/api/src/chan/http/HttpHolder.java
package chan.http;

import android.net.Uri;
import chan.library.api.BuildConfig;

public final class HttpHolder {
	public HttpHolder() {}

	public void disconnect() {
		BuildConfig.Private.expr();
	}

	public int getResponseCode() {
		return BuildConfig.Private.expr();
	}

	public Uri getRedirectedUri() {
		return BuildConfig.Private.expr();
	}

	public String getCookieValue(String name) {
		return BuildConfig.Private.expr(name);
	}

	public HttpResponse read() throws HttpException {
		return BuildConfig.Private.expr();
	}

	public void checkResponseCode() throws HttpException {
		BuildConfig.Private.<HttpException>error();
		BuildConfig.Private.expr();
	}
}

File: sdk/api/src/chan/http/HttpHolder.kt
package chan.http

import android.net.Uri
import chan.library.api.BuildConfig

class HttpHolder {
    fun disconnect() {
        BuildConfig.Private.expr<Any>()
    }

    fun getResponseCode(): Int = BuildConfig.Private.expr()

    fun getRedirectedUri(): Uri = BuildConfig.Private.expr()

    fun getCookieValue(name: String): String? = BuildConfig.Private.expr(name)

    @Throws(HttpException::class)
    fun read(): HttpResponse = BuildConfig.Private.expr()

    @Throws(HttpException::class)
    fun checkResponseCode() {
        BuildConfig.Private.error<HttpException>()
        BuildConfig.Private.expr<Any>()
    }
}
